src/ol/imagetile.js

```diff
@@ -94,6 +95,10 @@ ol.ImageTile.prototype.getKey = function() {
  */
 ol.ImageTile.prototype.handleImageError_ = function() {
   this.state = ol.TileState.ERROR;
+  var ctx = ol.dom.createCanvasContext2D(1, 1);
+  ctx.fillStyle = 'rgba(0,0,0,0)';
+  ctx.fillRect(0, 0, 1, 1);
+  this.image_ = ctx.canvas;
```

Comment:
We could also use a single static transparent image.  Maybe something like:

```js
ol.ImageTile.BlankImage = new Image();
ol.ImageTile.BlankImage.src = 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7';
```

And then use that image in the error handler.